plugins/modules/info.py

```diff
@@ -4003,6 +4003,13 @@ def get_support_assist_settings(self):
         else:
             return {}
 
+    def get_maintenance_settings(self):
```

Comment:
Would UT be updated to cover the change?

Reply:
Added.

Reply:
Please check https://github.com/dell/ansible-powerscale/pull/190#issuecomment-2968967239